apps/web/vibes/schema.ts

```diff
@@ -14,7 +14,7 @@ export type Component = {
   >
 }
 
-export type Components = NonEmptyArray<Component>
+export type Components = Array<Component>
```

Comment:
Do we want to support empty arrays for values? It seems that a VIBE would include at least one of these

Reply:
Ok, I'm seeing now that we're using this for "coming soon" VIBES. Maybe we want to have a separate type for that so we're not making the main type have to change to satisfy a "coming soon" VIBE? In particular since we don't need navigation, components, brands, tags, etc.